docs/docs/overview/performance.md

```diff
@@ -6,103 +6,178 @@ This section analyzes the performance of Constellation.
-The difference can largely be attributed to Constellation's [network encryption](../architecture/networking.md) that protects data in-transit.
+![Network Pod2Pod UDP benchmark graph](../_media/benchmark_net_p2p_udp.png)
 
-##### Intra-node
+##### Pod-to-Service
 
-Intra-node communication happens between pods running on the same node.
-The connections directly pass through the node's OS layer and never hit the network.
-The benchmark evaluates how the [Constellation's node OS image](../architecture/images.md) and runtime encryption influence the throughput.
+Tn this scenario, the client Pod connects to the server Pod via a ClusterIP service. This is more relevant to real-world use cases.
 
-Constellation's bandwidth for both sending and receiving is at 31 Gbps on Azure and 22 Gbps on GCP. AKS achieves 26 Gbps and GKE achieves about 27 Gbps in the tests.
+![Pod2Pod concept](../_media/benchmark_p2svc_concept.webp)
 
-![](../_media/benchmark_net.png)
+The results for “Pod-to-Service” TCP are as follows:
 
+![Network Pod2SVC TCP benchmark graph](../_media/benchmark_net_p2svc_tcp.png)
+
+The results for “Pod-to-Service” UDP are as follows:
+
+![Network Pod2SVC TCP benchmark graph](../_media/benchmark_net_p2svc_udp.png)
+
+Comparing Constellation on GCP with GKE, Constellation has 58% less TCP bandwidth less but a similar UDP bandwidth.
+Constellation on Azure isn't directly comparable as the Confidential VM types in Azure have only 10% of the regular VM types' bandwidth.
+Constellation's network encryption adds almost no overhead on top of that limitation.
 #### Storage I/O
 
 Azure and GCP offer persistent storage for their Kubernetes services AKS and GKE via the Container Storage Interface (CSI). CSI storage in Kubernetes is available via `PersistentVolumes` (PV) and consumed via `PersistentVolumeClaims` (PVC).
-Upon requesting persistent storage through a PVC, GKE and AKS will provision a PV as defined by a default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+Upon requesting persistent storage through a PVC, GKE, and AKS will provision a PV as defined by a default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/).
 Constellation provides persistent storage on Azure and GCP [that's encrypted on the CSI layer](../architecture/encrypted-storage.md).
-Similarly, Constellation will provision a PV via a default storage class upon a PVC request.
+Similarly, upon a PVC request, Constellation will provision a PV via a default storage class.
+Specifically, on Azure the [Azure Disk CSI driver with encryption](https://github.com/edgelesssys/constellation-azuredisk-csi-driver) v1.1.2 was used.
+On GCP the [Compute Engine Persistent Disk CSI Driver with encryption](https://github.com/edgelesssys/constellation-gcp-compute-persistent-disk-csi-driver) v1.1.2.
```

Comment:
These sentences seem a bit misplaced or lack context.

Reply:
Removed that entirely. It's fixed with the Constellation version anyway. So this is probably just confusing.